### ISSUE TITLE
LesbianX search results give SSL error

### ISSUE THREAD

**Issue by PAhelper** [PAhelper/PhoenixAdult.bundle#13]
I don't have the error in front of me, but it basically said liburl2 couldn't connect because no matching SSL cypher was found. I think that basically means the SSL cert for lesbianx.com is newer than the current python code can handle.

**Comment by fritz276** [comment]
URLError: <urlopen error [SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:590)>

It's the same with lubed.com
There are tons of hits with this error on the plex forum, the most interesting is maybe

https://forums.plex.tv/t/https-broken/216635

**Comment by pahelper3** [comment]
The problem keeps showing in the latest version. So does it means that EmpireX network simply doesn't work?

**Comment by cooltwa** [comment]
Small note to add, I've had success matching scenes from all XEmpire sites except LesbianX.

**Comment by mutluerol** [comment]
How can i resolve that problem ? is it depends plex python problem ?

**Comment by PAhelper** [comment]
So I've solved this issue for LesbianX in the latest code push. Basically I went around Lesbianx.com and instead use xempire.com to match and pull the scene metadata. This is only a workaround for this one site, not a solution unfortunately.

Yes, it has to do with the Python version used by Plex. Some of the functions built into Plex for Metadata agents to use are built on older Python functions that don't support newer SSL cyphers. As a result, any site that has a newer SSL certificate will fail to load when using these default Plex functions.
